alerts/monquorum.libsonnet

```diff
@@ -20,6 +20,22 @@
               severity_level: 'error',
             },
           },
+          {
+            alert: 'CephMonHighNumberOfLeaderChanges',
+            expr: |||
+              rate(ceph_mon_num_elections{%(cephExporterSelector)s}[15m]) > 3
+            ||| % $._config,
+            'for': $._config.monQuorumLeaderChangesAlertTime,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Ceph Monitor has seen many leader changes recently.',
```

Comment:
Lets avoid `Ceph` keyword in message. May be something like `Storage cluster has seen many leader changes recently`.

Reply:
:+1: fixed :)